google/gax/__init__.py

```diff
@@ -150,8 +163,9 @@ class RetryOptions(
     """Per-call configurable settings for retrying upon transient failure.
 
     Attributes:
-      retry_codes: a list of exceptions upon which a retry should be attempted.
-      backoff_settings: a BackoffSettings object configuring the retry
+      retry_codes (list[string]): a list of exception codes upon which a retry
```

Comment:
s/exception codes/status codes (at least in the gRPC terminology we're using). Is there a way we can link to the strings that are valid, i.e., keys into the dictionary in gRPC.py?